packages/react-scheduler/npm/umd/react-scheduler-tracking.js

```diff
@@ -0,0 +1,95 @@
+/**
+ * @license React
+ *
+ * Copyright (c) 2013-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+'use strict';
+
+(function(global, factory) {
+  typeof exports === 'object' && typeof module !== 'undefined'
+    ? (module.exports = factory(require('react')))
+    : typeof define === 'function' && define.amd // eslint-disable-line no-undef
+      ? define(['react'], factory) // eslint-disable-line no-undef
+      : (global.ReactSchedulerTracking = factory(global));
+})(this, function(global) {
+  function __getInteractionsRef() {
+    return global.React.__SECRET_INTERNALS_DO_NOT_USE_OR_YOU_WILL_BE_FIRED.SchedulerTracking.__getInteractionsRef.apply(
+      this,
+      arguments
+    );
+  }
+
+  function __getSubscriberRef() {
+    return global.React.__SECRET_INTERNALS_DO_NOT_USE_OR_YOU_WILL_BE_FIRED.SchedulerTracking.__getSubscriberRef.apply(
+      this,
+      arguments
+    );
+  }
+
+  function unstable_clear() {
+    return global.React.__SECRET_INTERNALS_DO_NOT_USE_OR_YOU_WILL_BE_FIRED.SchedulerTracking.unstable_clear.apply(
+      this,
+      arguments
+    );
+  }
+
+  function unstable_getCurrent() {
+    return global.React.__SECRET_INTERNALS_DO_NOT_USE_OR_YOU_WILL_BE_FIRED.SchedulerTracking.unstable_getCurrent.apply(
+      this,
+      arguments
+    );
+  }
+
+  function unstable_getThreadID() {
+    return global.React.__SECRET_INTERNALS_DO_NOT_USE_OR_YOU_WILL_BE_FIRED.SchedulerTracking.unstable_getThreadID.apply(
+      this,
+      arguments
+    );
+  }
+
+  function unstable_subscribe() {
+    // eslint-disable-next-line max-len
+    return global.React.__SECRET_INTERNALS_DO_NOT_USE_OR_YOU_WILL_BE_FIRED.SchedulerTracking.unstable_subscribe.apply(
+      this,
+      arguments
+    );
+  }
+
+  function unstable_track() {
+    return global.React.__SECRET_INTERNALS_DO_NOT_USE_OR_YOU_WILL_BE_FIRED.SchedulerTracking.unstable_track.apply(
+      this,
+      arguments
+    );
+  }
+
+  function unstable_unsubscribe() {
+    // eslint-disable-next-line max-len
+    return global.React.__SECRET_INTERNALS_DO_NOT_USE_OR_YOU_WILL_BE_FIRED.SchedulerTracking.unstable_unsubscribe.apply(
+      this,
+      arguments
+    );
+  }
+
+  function unstable_wrap() {
+    return global.React.__SECRET_INTERNALS_DO_NOT_USE_OR_YOU_WILL_BE_FIRED.SchedulerTracking.unstable_wrap.apply(
+      this,
+      arguments
+    );
+  }
+
+  return Object.freeze({
+    __getInteractionsRef: __getInteractionsRef,
```

Comment:
It would be nice to have some automatic check that verifies the list of exports matches the list of commonjs exports 1:1

Reply:
(I understand you added a fixture but someone could add a method to CJS API, and forget to test it in fixture)

Reply:
Yeah, that's reasonable. Although I think the bundled UMD solution is pretty nice, I'm not very happy with these "shims" and the way they kind of bypass our normal build system.

I'm not sure how to add an automated check without increasing the hackiness. Any ideas?

Reply:
A node script that imports all bundles after build and verifies their exports match up.

Or a Jest test if you can figure out a way to import the entry point shim.

Reply:
Whoops, this comment just came through. I'll tackle this with a follow up :)